subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/model/intention/State.kt

```diff
@@ -2,13 +2,18 @@ package com.avito.runner.service.model.intention
 
 data class State(
     val layers: List<Layer>,
-    val digest: String = layers.hashCode().toString()
+    val digest: String = layers.hashCode().toString() //layers order matter
```

Comment:
I think Layer is broken abstraction
Maybe we need simple properties: api, model etc?

Reply:
Totally agree, but the purpose of this pr is not related to refactor this abstraction thoroughly, this is a big deal.